CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Testify Change Log
 
+## Unreleased
+
+### Library
+
+#### Changes
+
+- rootViewId on ScreenshotRule is now public
```

Comment:
/nit this reflects the previous version. Maybe something like `rootViewId is exposed via getRootViewId() on ScreenshotRule`?